File: PSET1/mario.c
#include <cs50.h>
#include <stdio.h>

int get_height(string prompt);

int main(void)
{
    int n = get_height("Height: ");


    for (int i = 0; i < n; i++)
    {
        for (int j = 0; j < n - i - 1; j++)
        {
            printf(" ");
        }
        for (int j = 0; j < i + 1; j++)
        {
            printf("#");
        }
    printf("\n");
    }



}

// Prompt user for positive integer
int get_height(string prompt)
{
    int h;
    do
    {
        h = get_int("%s", prompt);
    }
    while (h < 1 || h > 8);
    return h;
}

File: PSET6/mario.py
from cs50 import get_int


while True: 
    height = get_int("Height: ")
    if height < 9 and height > 0:
        break

for i in range(height):
    
    for x in range(height - i - 1):
        print(" ", end="")
    
    for j in range(i + 1):
        print("#", end="")
    
    print("  ", end="")
    
    for j in range(i + 1):
        print("#", end="")
        
    print("")
